cg/cli/generate/report/utils.py

```diff
@@ -0,0 +1,121 @@
+"""Delivery report helpers"""
+
+import logging
+from datetime import datetime
+from typing import Optional
+
+import click
+from cgmodels.cg.constants import Pipeline
+
+from cg.constants import REPORT_SUPPORTED_PIPELINES
+from cg.meta.report.report_api import ReportAPI
+from cg.meta.report.balsamic import BalsamicReportAPI
+from cg.meta.report.balsamic_umi import BalsamicUmiReportAPI
+from cg.meta.report.mip_dna import MipDNAReportAPI
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.meta.workflow.balsamic_umi import BalsamicUmiAnalysisAPI
+from cg.meta.workflow.mip_dna import MipDNAAnalysisAPI
+from cg.store import models
+
+LOG = logging.getLogger(__name__)
+
+
+def resolve_report_case(context: click.Context, case_id: str) -> models.Family:
```

Comment:
Why “resolve” and not “get” since you are returning something that is supposed to be used later?